projects/1.16/assets/tips/tips/lang/zh_cn.json

```
@@ -1 +1,72 @@
+  "tips.tip.poison_immune": "僵尸、骷髅和蜘蛛免疫剧毒药水。",
+  "tips.tip.witch_conversion": "被闪电击中的村民将变成女巫。",
+  "tips.tip.four_eggs": "抛出鸡蛋时，有极小概率生成4只小鸡。",
+  "tips.tip.nether_lava": "熔岩在下界的流动速度会更快。",
+  "tips.tip.east": "太阳、月亮和星星都向东方移动。",
+  "tips.tip.swamp_slime": "史莱姆会在满月时的沼泽中生成。",
+  "tips.tip.deflect_ghast_fireballs": "你能够通过攻击恶魂发出的火焰弹来改变它的运动方向。",
+  "tips.tip.shulker_armor": "潜影贝在打开时会受到更多的伤害。",
+  "tips.tip.arthropods": "节肢杀手能够对蜘蛛、蠹虫、蜜蜂、末影螨造成额外伤害。",
+  "tips.tip.fossils": "在沼泽和沙漠下能够发现大型化石。",
+  "tips.tip.crouch_blocks": "潜行/蹲下可以防止你从方块边缘掉下来。",
+  "tips.tip.smoke_bees": "在蜂巢下方放置营火会使蜜蜂不容易被激怒。",
+  "tips.tip.lava_fuel": "熔岩桶可以作为熔炉的燃料使用。",
+  "tips.tip.wooden_tool_fuel": "不再使用的木制工具能够为熔炉燃料使用。",
+  "tips.tip.scrap_nuggets": "金制与铁制的工具/盔甲能够冶炼成金/铁粒",
+  "tips.tip.piglin_gold": "猪灵是真的非常喜欢黄金。",
+  "tips.tip.dog_health": "狼的尾巴越低，它的剩余血量就越少。",
+  "tips.tip.dog_rotten_flesh": "你可以通过喂狼吃腐肉来恢复它的血量。",
+  "tips.tip.lure_animals": "你可以在主手/副手拿上动物喜欢食物来引诱动物跟随你。",
+  "tips.tip.cooked_drops": "燃烧死亡的动物有时会掉落熟肉。",
+  "tips.tip.nether_sponge": "在下界放置的湿海绵会瞬间干燥。",
+  "tips.tip.boat_passenger": "船只能承载两人。",
+  "tips.tip.shulker_golem": "根据游戏代码，潜影贝被视为一种傀儡。",
+  "tips.tip.note_block_instrument": "一个音符块的播放的乐器是由它下方的方块决定的。",
+  "tips.tip.zombie_armor": "即使僵尸没有穿戴任何护甲，他们也拥有两点基础护甲值。",
+  "tips.tip.undead_helmet": "带着头盔的亡灵生物在白天不会燃烧。",
+  "tips.tip.iron_golem_heal": "铁傀儡可以用铁锭来治疗。",
+  "tips.tip.turtle_shell": "杀死海龟并不会掉鳞甲，但小海龟长大时会掉。",
+  "tips.tip.animal_growing": "您可以喂养小动物来促进它们的成长。",
+  "tips.tip.blaze_snowball": "烈焰人是唯一会受到雪球伤害的敌对生物。",
+  "tips.tip.biome_bonemeal": "在草块上使用骨粉，会根据你所在的生物群系给你不同的花。",
+  "tips.tip.biome_fishing": "与在其他生物群系中钓鱼相比，在丛林中钓鱼可以给您不同的物品。",
+  "tips.tip.fishing_resources": "钓鱼是获得高等级的附魔书的好方法。",
+  "tips.tip.boat_jump": "通过在熔岩中放置一艘船，您可以制作一个临时平台。",
+  "tips.tip.creeper_cat": "苦力怕害怕猫，如果太靠近猫会逃跑。",
+  "tips.tip.water_grass": "水流可以快速破坏草并掉落种子。",
+  "tips.tip.gravity_torch": "如果你手速够快，可以通过放火把在受重力影响的方块下面来破坏一串的这些方块。",
+  "tips.tip.string_wool": "4个线可以合成羊毛。",
+  "tips.tip.barrel": "木桶比箱子更划算，它比箱子少用一个木板，同时可以无限地放在一起。",
+  "tips.tip.job_site_dimension": "村民改变所在维度时会失去工作地点。",
+  "tips.tip.villager_sweat": "村民在突袭中试图与他们交易时会出汗（粒子）。",
+  "tips.tip.wandering_invis": "流浪商人在晚上喝隐形药水。",
+  "tips.tip.baby_panda_slimeball": "熊猫宝宝打喷嚏后有0.1%的几率掉落黏液球。",
+  "tips.tip.fortress_triangulation": "你可以通过从两个不同的位置使用末影之眼并通过对要塞位置进行三角定位来寻找它们相交的位置。",
+  "tips.tip.creeper_drops_disc": "如果骷髅或流浪者的箭杀死了苦力怕，苦力怕会随机掉落一张音乐唱片（不包括下界音乐唱片）。",
+  "tips.tip.mooshroom_lightning": "红色哞菇被闪电击中时会变成棕色哞菇，反之亦然。",
+  "tips.tip.bed_explode": "试图在其他维度睡觉会很糟糕。",
+  "tips.tip.fall_damage_blocks": "一些像水和干草块这样的方块可以减少摔落伤害。",
+  "tips.tip.bed_respawn": "在床上睡觉会设置你的出生点。",
+  "tips.tip.ore_tiers": "有些矿石有最低要求的挖掘等级来开采它们。",
+  "tips.tip.sponge_bucket": "湿海绵可以通过在熔炉中烧炼来干燥。如果燃料槽中有空桶，桶会接水！",
+  "tips.tip.coarse_to_dirt": "砂土可以使用锄头转化为普通泥土。",
+  "tips.tip.enderman_blocks": "末影人可以移动泥土、沙子和TNT等方块！",
+  "tips.tip.dragon_blocks": "铁栏杆、末地石和黑曜石免疫末影龙的破坏。",
+  "tips.tip.piglin_repellent": "猪灵害怕灵魂火，包括灵魂火把和灵魂灯笼。",
+  "tips.tip.portal_trick": "通过使用水桶和熔岩桶，你可以制作一个简单的下界传送门。",
```

```suggestion
  "tips.tip.portal_trick": "通过使用水桶和熔岩桶，你可以轻松建造一个下界传送门。",
```
这里的easy是表示“建造此下界门很方便很轻松（不用镐子挖半天）”。